lib/index.js

```diff
@@ -17,6 +17,25 @@ var shell = require('shelljs');
 var restify = require('restify');
 
 /**
+ * checks if the path is device rooted, that is, if it has [drive letter]:\ at its beggining.
+ * @private
+ * @param {String} fpath
+ *      The file path to check
+ * @returns {Boolean}
+ *      Whether the path is rooted or not.
+ */
+function isWin32DeviceRoot(fpath) {
+    // Only for win32
+    if (process.platform !== 'win32') {
+        return false;
+    }
+
+    // Any drive letter followed by :\
+    var deviceRoot = /^[A-Za-z]:\\/;
+    return deviceRoot.test(fpath);
```

Comment:
The reason why I don't use `path.isAbsolute` is because it would create trouble if the string was a unix string or a UNC. I must have a drive letter (*absolute device path*) or I won't do it.

That's also the reason why I test for win32; I don't even try checking a drive letter if I'm on linux. 

Check the image below; I'm on Windows 7 running Node 6.1. 

![image](https://cloud.githubusercontent.com/assets/1890900/17109750/b1bad1e0-526f-11e6-808b-ec5a3b6e6012.png)